scanners/generic/tools/oobtkube.py

```diff
@@ -132,6 +132,15 @@ def find_leaf_keys_and_test(
     Iterate the spec data and test each parameter by modifying the value with the attack payload.
     Test cases: appending 'curl' command, TBD
     """
+    try:
```

Comment:
Nice addition. BTW, how about checking this in the main() and just exit in case of error?